ModelLibrary/Utility/ExternalWebMethod.cs

```diff
@@ -0,0 +1,204 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Xml.Linq;
+using VAdvantage.DataBase;
+using VAdvantage.Model;
+
+namespace VAdvantage.Utility
+{
+    class ExternalWebMethod
```

Comment:
access modifier should be public.

Reply:
Sir ye internal (default access of class) nai chlega ?